package.json

```diff
@@ -13,14 +13,21 @@
   },
   "scripts": {
     "build": "babel src --out-dir dist",
+    "lint": "eslint src/**/*.js",
     "test": "babel-node test/specs/main"
   },
+  "dependencies": {
+    "eslint": "^4.19.1",
```

Comment:
Shouldn't this be under `devDependencies`? same as `husky`? they're not needed for this project to actually work when other people install it under their projects